docs/photoniq/vwrs/01-how-vwrs-works.md

```diff
@@ -0,0 +1,43 @@
+---
+title: How VWRs Works
+---
+
+Macrometa PhotonIQ Virtual Waiting Rooms (VWRs) are an advanced prioritization service that manages the flow of visitors entering a site, especially during periods of high demand. It serves as a protective layer for your origin server, effectively managing high traffic volumes to prevent server overload.
+
+## Solution Overview
+
+![VWR Solution Architecture](../../../static/img/photoniq/vwr/vwr-solution-architecture.png)
+
+The waiting room solution includes these essential aspects:
+
+- **Origin Service**: The website for which you want to set up a virtual waiting room.
+- **Origin Metrics Service**: Allows you to retrieve information about a specific domain.
+- **Waiting Room Page**: The endpoint (URL) with the waiting room attached.
```

Comment:
I dont understand.  An endpoint that has waiting room attached.
Is attached the right word?